src/Mvc/Mvc.ApiExplorer/src/EndpointMetadataApiDescriptionProvider.cs

```diff
@@ -234,6 +235,25 @@ private ApiDescription CreateApiDescription(RouteEndpoint routeEndpoint, string
             }
         }
 
+        private static void AddAcceptsRequestFormats(
+            IList<ApiRequestFormat> supportedRequestFormats,
+            EndpointMetadataCollection endpointMetadata)
+        {
+            var requestMetadata = endpointMetadata.GetOrderedMetadata<IAcceptsMetadata>();
+            var declaredContentTypes = GetAcceptsContentTypes(requestMetadata);
```

Comment:
It looks like we're missing the bit where we set a ApiParameterDescription here based on `IAcceptsMetadata.RequestType`. I'm 🆗 getting it in for rc2 if it means we can get this in for rc1

Reply:
Yes,  I have moved that part to a different issue.

Reply:
Where is this being tracked?

Reply:
https://github.com/dotnet/aspnetcore/issues/35421